common/static/common/templates/discussion/forum-action-report.underscore

```diff
@@ -1,5 +1,5 @@
 <li class="actions-item">
-    <a href="javascript:void(0)" class="action-list-item action-report" role="checkbox" aria-checked="false">
+    <button class="btn-link action-list-item action-report" role="checkbox" aria-checked="false">
```

Comment:
Is there JavaScript keeping tabs of the new role here? Meaning, when this is pressed, does `aria-checked` change its value accordingly? Any reason not to use an actual checkbox input type here?